### PR TITLE
updated HbarChart style

### PR DESCRIPTION
**Pull request by Nastia1409**
changed the hbarChart style #147 

image

### CODE REVIEW

File: src/pages/dashboard/HbarChart/HbarChart.tsx [hunk 1]
```diff
@@ -16,43 +16,51 @@ export function HbarChart({
...
     ...entry,
   }))
   console.log(rows[0])
 
+  const wrapWithTooltip = (element: React.ReactElement, entry: Entry) => {
+    return (
+      <Tooltip
+        placement={'top'}
+        title={
+          <div style={{ fontSize: 15 }}>
```
Comment (NoamGaash) [inline src/pages/dashboard/HbarChart/HbarChart.tsx:31]: can be moved to the CSS file

**Review by NoamGaash — APPROVED** [review]
I LOVE your coding style!
Thanks for your contribution :partying_face:

**Comment by Nastia1409** [comment]
@NoamGaash thanks!
then you can merge it, right?
seems like I don't have the right access to merge

**Comment by NoamGaash** [comment]
@Nastia1409 I had a [suggestion](https://github.com/hasadna/open-bus-map-search/pull/152#discussion_r1369677668)

**Pull request merged**